test/functional/legacy/032_insert_expansion_spec.lua

```diff
@@ -0,0 +1,90 @@
+-- Test for insert expansion
+-- :se cpt=.,w
+-- * add-expands (word from next line) from other window
+-- * add-expands (current buffer first)
+-- * Local expansion, ends in an empty line (unless it becomes a global expansion)
+-- * starts Local and switches to global add-expansion
+-- :se cpt=.,w,i
+-- * i-add-expands and switches to local
+-- * add-expands lines (it would end in an empty line if it didn't ignored it self)
+-- :se cpt=kXtestfile
+-- * checks k-expansion, and file expansion (use Xtest11 instead of test11,
+-- * because TEST11.OUT may match first on DOS)
+-- :se cpt=w
+-- * checks make_cyclic in other window
+-- :se cpt=u nohid
+-- * checks unloaded buffer expansion
+-- * checks adding mode abortion
+-- :se cpt=t,d
+-- * tag expansion, define add-expansion interrupted
+-- * t-expansion
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('Insert expansion', function()
+  setup(clear)
+
+  it('is working', function()
+    insert([[
+      start of testfile
+      run1
+      run2
+      end of testfile
+
+      test11	36Gepeto	/Tag/
+      asd	test11file	36G
+      Makefile	to	run]])
+
+    execute('se backspace=""')
+    execute('se cpt=.,w ff=unix | $-2,$w!Xtestfile | set ff&')
+    execute('se cot=')
+    feed('<C-W>nO#include "Xtestfile"<cr>')
+    feed('ru<C-N><C-N><C-X><C-N><esc><C-A><cr>')
```

Comment:
Possibly the `<esc><c-a>` is being interpreted as "meta ctrl-a" chord. Need to break it up.